docs/docs/basics/00-app-anatomy.md

```diff
@@ -104,7 +104,7 @@ In general, the `InitChainer` is mostly composed of the [`InitGenesis`](../build
 See an example of an `InitChainer` from `simapp`:
 
 ```go reference
-https://github.com/cosmos/cosmos-sdk/blob/v0.47.0-rc1/simapp/app.go#L569-L577
+https://github.com/cosmos/cosmos-sdk/blob/v0.50.0-alpha.0/simapp/app.go#L569-L577
 ```
 
 ### BeginBlocker and EndBlocker
```

Comment:
This should be updated. It should be clear that BeginBlocker and EndBlocker are something app only and not ABCI releated.